config/settings.yml

```diff
@@ -1,3 +1,5 @@
 moab:
-  storage_roots: 'spec/fixtures'
-  storage_trunk: 'moab_storage_root'
+  # storage_roots: 'spec/fixtures'
+  # storage_trunk: 'moab_storage_root'
+  storage_roots: ['/services-disk11']
+  storage_trunk: 'sdr2objects'
```

Comment:
when the time comes to take off WIP, your local settings should NOT be included.  The actual way to do this is with a file `config/settings.local.yml` file -- the settings in that file take precedence over these, and we have this in .gitignore:

```
config/settings.local.yml
config/settings/*.local.yml
config/environments/*.local.yml
```